app/views/cars/index.html.erb

```diff
@@ -1,10 +1,16 @@
 <h1>Cars</h1>
 
-<div class="card" style="width: 18rem;">
-  <img class="card-img-top" src="..." alt="Card image cap">
-  <div class="card-body">
-    <h5 class="card-title"><%= @cars.first.title %></h5>
-    <p class="card-text"><%= @cars.first.description %></p>
-    <a href="#" class="btn btn-primary">View More</a>
+<% @cars.each do |car| %>
+  <div class="card" style="width: 18rem;">
+    <% if element.photo.attached? %>
+      <img class="card-img-top" salt="Card image cap"><%= cl_image_tag element.photo.key %></img>
+    <% end %>
+    <div class="card-body">
+      <h5 class="card-title"><%= @cars.title %></h5>
+      <h5 class="card-title"><%= @cars.year %></h5>
+      <h5 class="card-title"><%= @cars.price %></h5>
+      <p class="card-text"><%= @cars.description %></p>
```

Comment:
We don't want to display the whole info in our index page. We have the show page for that.